src/index.css

```diff
@@ -19,14 +19,6 @@
   @apply my-2 block text-left text-sm font-bold text-gray-700 dark:text-white;
 }
 
-.input-underline {
```

Comment:
I think the css classes are still used somewhere, so deleting them now wouldn't be a good idea IMO.

Reply:
True. I guess it depends on the scope or way of refactoring.
I assume we won't be using any of these in the future design. And I assume we won't have state of "in between designs". It will all be done step by step and then a new release will update the full design.
wdyt?

Reply:
We do have a "in-between" step by only redesigning the setup first. :)

Reply:
Ah, and that will go live then? Got it